pkg/stores/proxy/proxy_store.go

```diff
@@ -361,12 +361,22 @@ func (s *Store) WatchNames(apiOp *types.APIRequest, schema *types.APISchema, w t
 	go func() {
 		defer close(result)
 		for item := range c {
+			if item.Type == watch.Error {
+				if status, ok := item.Object.(*metav1.Status); ok {
+					logrus.Debugf("WatchNames received error: %s", status.Message)
+				} else {
+					logrus.Debugf("WatchNames received error: %s", item)
```

Comment:
I don't think watch.Event has a `String()` so this likely would have failed a `go vet` and will result in an improperly formatted log message. Did you want `%v` instead of `%s`?

Reply:
It looks like the CI validate step is broken at the moment?

Reply:
Right on both accounts. Fixed in https://github.com/rancher/steve/pull/109 and https://github.com/rancher/steve/pull/110 for master, v2.7-debug-41225-1 fixed manually